developer/src/kmc-ldml/src/compiler/compiler.ts

```diff
@@ -146,7 +153,7 @@ export class LdmlKeyboardCompiler implements KeymanCompiler {
     const kmx_binary = builder.compile();
 
     const vkcompiler = new LdmlKeyboardVisualKeyboardCompiler(this.callbacks);
-    const vk = vkcompiler.compile(source);
+    const vk = vkcompiler.compile(kmx.kmxplus, this.callbacks.path.basename(outputFilename, '.kmx'));
```

Comment:
```suggestion
    const vk = vkcompiler.compile(kmx.kmxplus, outputFilename.replace(/\.kmx$/, ''));
```

Matches pattern for setting `outputFilename` -- we could add a `baseFilename` variable instead...